### ISSUE TITLE
[Component] Number input

### ISSUE THREAD

**Issue by flore2003** [Rocketloop/ng-rocketparts#2]
We need to extract our number input and expose it as part of this library. We also need to create tests for the component, before it can be included in the library.

**Comment by ditoglez** [comment]
In order for the component to work in the library I had to provide the `DecimalPipe` in component's providers. After that, it seems to work correctly with  template-driven and reactive forms.

**Comment by ditoglez** [comment]
Updated the component to use `formatNumber` instead of `DecimalPipe`

**Comment by ditoglez** [comment]
@flore2003, @erzaehlsalex, I think this should be provided as an `NgModule` in case someone would like to use it standalone. What do you think guys?

**Comment by flore2003** [comment]
Yes, I think that is a good idea, let's do it like ng-bootstrap does it: One NgModule per Component/Feature and then one top-level NgModule for the whole library as a convenience module that imports all other modules.
